xinference/client/restful/restful_client.py

```diff
@@ -829,15 +829,11 @@ def launch_model(
         n_gpu: Optional[Union[int, str]],
             The number of GPUs used by the model, default is "auto".
             ``n_gpu=None`` means cpu only, ``n_gpu=auto`` lets the system automatically determine the best number of GPUs to use.
+        mode_deft_config: Optional[PeftModelConfig],
+            The config for the lora model.
```

Comment:
A more detailed description of how this data structure should be set up in the case of multi lora